src/main/kotlin/io/sip3/salto/ce/mongo/MongoCollectionManager.kt

```diff
@@ -82,6 +83,11 @@ class MongoCollectionManager : AbstractVerticle() {
                     if (!names.contains(name)) {
```

Comment:
Let's make it easier:
1. If if (!names.contains(name)) { createCollection }
2. Separately - createIndexes